Parse/src/main/java/com/parse/ParseObject.java

```diff
@@ -98,6 +99,7 @@ private static ParseObjectSubclassingController getSubclassingController() {
       private long createdAt = -1;
       private long updatedAt = -1;
       private boolean isComplete;
+      private Set<String> safeKeys = new HashSet<>();
       /* package */ Map<String, Object> serverData = new HashMap<>();
```

Comment:
Haha yes. Actually the server does not return these, we have them because of query.selectKeys(), but you got the point. Annoying bug.